app/script/entity/message/ContentMessage.js

```diff
@@ -117,7 +117,11 @@ z.entity.ContentMessage = class ContentMessage extends z.entity.Message {
   }
 
   isReplyToUser(userId) {
-    return this.quote() && this.quote().isQuoteFromUser(userId);
+    return this.quote() ? this.quote().isQuoteFromUser(userId) : false;
+  }
+
+  isResponseToUser(userId) {
```

Comment:
Not sure about the naming of this function. 
`isReply` and `isResponse` are very close and hard to distinguish. 

I believe we can come up with a better name. 

WDYT of `isTargetingUser`?